src/main/java/org/joda/money/BigMoney.java

```diff
@@ -1618,7 +1618,7 @@ public int compareTo(BigMoneyProvider other) {
      * The compared values must be in the same currency.
      * 
      * @param other  the other monetary value, not null
-     * @return true is this is greater than the specified monetary value
+     * @return true iff this is equal to the specified monetary value

```

Comment:
While I know what "iff" means, it isn't common knowledge. Can you change it to a plain "if" please.